Add release and LTS guides (#100)
humanmade/altis-documentation: other-docs/guides/long-term-support.md

# Long Term Support

To provide stability to Altis users, major Altis releases will be supported for a minimum of one year after their release. The latest release of Altis will receive full support, while older releases will receive support under the "long-term support" policy.

[Learn more about how Altis releases work here](./altis-releases.md).

> At any one time, only a single major Altis version is fully supported, and is the "active" release. When a new major version of Altis is published, the previously active release moves into long-term support mode.

Reviewer: ```suggestion
At all times, only a single major Altis version is fully supported, and is the "active" release. When a new major version of Altis is published, the previously active release moves into long-term support mode.
```
